wgengine/magicsock/magicsock.go

```diff
@@ -2930,13 +2901,13 @@ func (c *Conn) rebind(curPortFate currentPortFate) error {
 	if runtime.GOOS == "js" {
 		return nil
 	}
+	if err := c.bindSocket(&c.pconn6, "udp6", curPortFate); err != nil {
```

Comment:
okay, I understand why this is safe now: all the failure paths from bindSocket do `ruc.setConnLocked(newBlockForeverConn())` so other calls to the Rebinding conn won't crash later.